### ISSUE TITLE
pt_BR locale not working

### ISSUE THREAD

**Issue by rhenanbartels** [Pierre-Sassoulas/django-survey#106]
Hi,  I would like to thank you all for developing this package. It saved me a lot of time. Congratulations on the great work!

I've noticed that the `pt_BR/pt-br` is not working. 

I've checked the `django-survey/survey/locale/pt-br/LC_MESSAGES/` folder and the `django.mo` file is missing. I've also checked the `survey/settings.py` and the variable `LAGUAGES` is missing the ("pt-br", "Brazilian Portuguese") option.

Thank you very much,

Rhenan

**Comment by Pierre-Sassoulas** [comment]
Hello, thank you for the bug report and for the kind words, much appreciated. Are you using the pypi package or the github repository directly ? What happens if you do `python manage.py compilemessages` for your project ?

**Comment by rhenanbartels** [comment]
Hi, I've installed the package using `pip install django-survey-and-report`.

If I run `python manage.py compilemessages`

I receive: 
```
CommandError: This script should be run from the Django Git checkout or your project or app tree, or with the settings module specified.```

**Comment by Pierre-Sassoulas** [comment]
Should be fixed in `1.3.28` :)

**Comment by rhenanbartels** [comment]
Perfect! Thank you very much

salutations du Brésil []s
